test/relayserver/RelayServer.test.ts

```diff
@@ -168,15 +177,12 @@ describe('RelayServer', function () {
 
     describe('validateInputTypes', function () {
       it('should throw if relayHub is undefined', async function () {
-        const userDefinedRelayRequest = createUserDefinedRequest(
-          IS_DEPLOY_REQUEST,
-          {
-            from: owner.address,
-            to: recipient.address,
-            data: encodedData,
-            nonce: 1,
-          }
-        );
+        const userDefinedRelayRequest = createUserDefinedRequest(false, {
```

Comment:
I know you didn't introduce this change, but what do you think if we create 2 functions `createDeployUserDefinedRequest` and `createRelayUserDefinedRequest` instead of using the flag?

Reply:
updated